src/main/scala/search/Constraints.scala

```diff
@@ -0,0 +1,23 @@
+package basic.search
+
+import gem.math.{ Angle, Wavelength }
+
+sealed trait Constraints
+object Constraints {
+
+  final case class Spectroscopy(
+    wavelength:         Wavelength,
+    wavelengthCoverage: Wavelength     = Wavelength.fromAngstroms.getOption(2000).get, // TODO: add fromNanometers
```

Comment:
`Option.get`?

Reply:
You can't have a negative wavelength so that's where the `Option` comes in.  I guess it could be `Wavelength.fromAngstroms.unsafeGet(2000)`.

Reply:
Yeah I forgot about `unsafeGet`, I'll fix that.